Rakefile

```diff
@@ -35,6 +36,19 @@ task :swift_doc_update do
   system("rm .swift-doc.version && echo \"#{SWIFTDOC_VERSION}\" >> .swift-doc.version")
 end
 
+desc("Updates swift-lint binary with the latest version available.")
+task :swift_lint_update do
+  root_dir = Dir.pwd.strip
+  Dir.mktmpdir do |temporary_dir|
+    Dir.chdir(temporary_dir) do
+      system("curl", "-LO", "https://github.com/realm/SwiftLint/releases/download/#{SWIFTLINT_VERSION}/portable_swiftlint.zip")
+      extract_zip("portable_swiftlint.zip", "portable_swiftlint")
+      system("cp", "portable_swiftlint/swiftlint", "#{root_dir}/vendor/swiftlint")
+    end
+  end
+  system("rm .swift-lint.version && echo \"#{SWIFTLINT_VERSION}\" >> .swift-lint.version")
+end
+
```

Comment:
@pepibumur this kind of changes shouldn't be included in `CHANGELOG.md` am I right?

Reply:
Yes. We can print something reminding people to update the CHANGELOG. If we have to do this very often we might consider automating it in the future.

Reply:
`swift_doc_update` and `swift_lint_update` have been updated